### PR TITLE
Import: Windows Server 2019

### PR DESCRIPTION
**Pull request by rofuentes**
- Import (translate) workflow
- Daisy integration test
- Test import image (no drivers/GCE binaries) exists in test project

### CODE REVIEW

File: daisy_workflows/image_import/windows/translate_windows_81_byol.wf.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        }
+      },
+      "Timeout": "60m"
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "${image_name}",
+          "SourceDisk": "${source_disk}",
+          "Licenses": ["projects/windows-cloud/global/licenses/windows-server-2012-r2-byol"],
```
Comment (adjackura) [inline daisy_workflows/image_import/windows/translate_windows_81_byol.wf.json]: Do we not have a windows-8-1-enterprise-byol license?

Comment (rofuentes) [reply daisy_workflows/image_import/windows/translate_windows_81_byol.wf.json]: This was probably an oversight, I will update and create the licence if it does not exist.

Comment (rofuentes) [reply daisy_workflows/image_import/windows/translate_windows_81_byol.wf.json]: I am moving the Windows 8.1 import workflow into a separate PR.

**Pull request merged**